metrics/default_metrics/tarantool/memory.lua

```diff
@@ -0,0 +1,18 @@
+local utils = require('metrics.default_metrics.tarantool.utils');
+
+local function get_memory_metrics()
+    if not utils.box_is_configured() then
+        return
+    end
+
+    if box.info.memory ~= nil then
+        local i = box.info.memory()
+        for k, v in pairs(i) do
+            utils.set_gauge('info_memory_' .. k, 'Memory' .. k, v)
+        end
+    end
+end
+
+return {
+    init = get_memory_metrics
```

Comment:
Maybe `update` instead of `init`?

We usually use init to configure a module for later usage. Think of it as a constructor.

This module doesn't have any state to configure.